test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -254,7 +254,8 @@ private static Person generateTestPerson() {
                 new Phone(Phone.EXAMPLE, true),
                 new Email(Email.EXAMPLE, false),
                 new Address(Address.EXAMPLE, true),
-                new UniqueTagList(new Tag("tag1"), new Tag("tag2"), new Tag("tag3"))
+                new UniqueTagList(new Tag("tag1"), new Tag("tag2"), new Tag("tag3")),
+                Person.nextSequenceNumber++
```

Comment:
This can be avoided if the sequence number was incremented inside the Person constructor.